ONSdigital/response-operations-ui
# Bug missing icons

# What and why?
Business review noticed Add icon not showing when adding CIs
I also noticed Download icons not showing (Report CSV) in CE table on survey.html

# How to test?
Check remove icon is showing: 
![image](https://user-images.githubusercontent.com/1080383/117831096-2ca2a200-b26c-11eb-81e0-263735efe175.png)

Check download icon is showing when CE is live:
![image](https://user-images.githubusercontent.com/1080383/117831318-58258c80-b26c-11eb-843b-79f37a221fac.png)

# Trello
https://trello.com/c/WtF3qpXr/673-bug-button-to-remove-collection-instruments-in-eq-is-not-visible-in-pre-prod
